microsetta_interface/templates/kits.jinja2

```diff
@@ -203,6 +203,11 @@
             {{ _('Thank you for logging your sample information. It looks like you haven\'t updated your profile recently. Please review') }} <a href="/accounts/{{account_id}}/sources/{{source_id}}/take_survey?survey_template_id={{prompt_survey_id}}">{{ _('your survey responses') }}</a> {{ _('to ensure they\'re as current and complete as possible.') }}
         </div>
         {% endif %}
+        {% if prompt_skin_app %}
+        <div class="alert alert-primary alert-nutrition mt-4" role="alert">
```

Comment:
Minor but this isn't nutrition?

Reply:
Adjusted, and adjusted the other prompt from which this was copied.